### DISCUSSION TITLE
Was wondering if there are any predefined profiles I can import?

### DISCUSSION THREAD

**Discussion by Raj2032** [wwmm/easyeffects#3160] [Q&A]
I am using headphones and was wondering I suppose I need to use `Equalizer`:

![image](https://github.com/wwmm/easyeffects/assets/29293830/67a3b84b-aaab-40c2-8da2-d6b83ebfd222)

and if so, are there any preset profiles or something I can import?

**Comment by wwmm** [comment]
We do not provide presets. But there are some links in our wiki to presets provided by the community https://github.com/wwmm/easyeffects/wiki/Community-Presets.

In general presets are just a starting point. Some customization by the user is often necessary. Depending on you headphone model it may be interesting to search in the internet for "AutoEq" or "Oratory" presets made for your headphone model. They use a preset format made for the equalizer APO used on Windows but as long as the preset is not too elaborated they can be loaded in EasyEffects through the "APO import" button.
